app/views/content/life-as-a-teacher/explore-subjects/history/_article.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach history</h2>
+    <div class="inset">
+      <p>Teaching history, you’ll challenge pupils to scrutinise evidence to build a picture of the past. Guiding them through the often elusive sources left to us, you will help them question the reliability and accuracy of what they read. They’ll build skills in research, handling evidence, and forming persuasive arguments.</p>
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching history opens the door for your pupils to pursue exciting careers in fields such as research, curation, and law.</p>
+      <p>Your pupils will learn more from you than you might think. As their history teacher, you'll develop your pupils’ knowledge and skills to investigate the past by interrogating primary and secondary sources.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_history_jack' %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-history-programmes-of-study/national-curriculum-in-england-history-programmes-of-study">national curriculum for history</a>.</p>
+    <p>When you teach 11 to 14 year olds (key stage 3), themes you may study include:</p>
+    <ul>
+      <li>the development of church, state and society</li>
+      <li>ideas, political power, industry and empire: Britain, 1745-1901</li>
+      <li>challenges for Britain, Europe and the wider world from 1901 to the present day</li>
+      <li>a local history study</li>
+      <li>a study of a significant society or issue in world history and its interconnections with other world developments</li>
+    </ul>
+    <p>Knowledge and skills you'll cover when you teach 14 to 16 year olds (key stage 4) include :</p>
+    <ul>
+      <li>developing and extending knowledge of key historical events</li>
+      <li>developing questioning about the past through the use of sources</li>
```

Comment:
Should this be questions?

Reply:
or just `questioning the past through the use of sources` ?